dbt_schema_builder/builder.py

```diff
@@ -191,33 +210,48 @@ def validate_schema_config(config):
         invalid schemas from being built
         """
         valid_keys = ['EXCLUDE', 'INCLUDE', 'SOFT_DELETE']
-        for _, app_config in config.items():
-            for schema, schema_config in app_config.items():
+        database_schema_pattern = re.compile(r'^[A-Za-z0-9_$]+\.[A-Za-z0-9_$]+$')
```

Comment:
Are we just planning to enforce the requirements for unquoted identifiers rather than quoted?

> The identifier can contain and even start with any ASCII character from the blank character (32) to the tilde (126).

https://docs.snowflake.com/en/sql-reference/identifiers-syntax.html

Reply:
Yes. To be honest, the main thing I'm trying to enforce is that there are two sections to this key, separated by a `.`.